lib/ougai/logger.rb

```diff
@@ -11,8 +11,8 @@ class Logger < ::Logger
 
     attr_accessor :default_message, :exc_key
 
-    def initialize(*args)
-      super(*args)
+    def initialize(*, **)
```

Comment:
memo) The arguments are composed of optional ones and keyword ones.  https://github.com/ruby/logger/blob/master/lib/logger.rb#L380